tedana/metrics/collect.py

```diff
@@ -61,7 +61,7 @@ def generate_metrics(
 
     if metrics is None:
         metrics = ["map weight"]
-    RepLGR.info("The following metrics were calculated: {}.".format(", ".join(metrics)))
+    RepLGR.info("\n\nThe following metrics were calculated: {}.".format(", ".join(metrics)))
```

Comment:
Probably not worth systematically going through all the code, but is it worth changing from `.format()` to f strings whenever we edit lines?

Reply:
I was planning to go hog-wild with style changes in #970 once the more substantive PRs were merged.